install/ontology/general.rdf

```diff
@@ -2,11 +2,11 @@
 <rdf:RDF
         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
-        xmlns:lom="http://www.tao.lu/Ontologies/lom.rdf#"
-        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
-        xml:base="http://www.tao.lu/Ontologies/lom.rdf#"
-        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
-        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:lom="http://www.taotesting.com/ontologies/lom.rdf#"
+        xmlns:tao="http://www.taotesting.com/ontologies/TAO.rdf#"
+        xml:base="http://www.taotesting.com/ontologies/lom.rdf#"
+        xmlns:generis="http://www.taotesting.com/ontologies/generis.rdf#"
```

Comment:
I will be difficult to make it work now ;-) ! My request was only for the LOM Ontology !